appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java

```diff
@@ -159,6 +166,37 @@ public VmApiProxyDelegate() {
           "complete. Service bridge status code: %s; response " +
           "content-length: %s. Took %s ms.", packageName, methodName, requestData.length, 200,
           responseData.length, (end - start)));
+      
+      // HACK TO FIX USER_SERVICE ISSUE #164
+      if ("user".equals(packageName)) {
+        boolean userservicefix = !Boolean.getBoolean("UserService164");
```

Comment:
Well it is not exactly "Local", as we still call the RPC, we just correct the results.   Perhaps UserServiceLocalSchemeHost?